tasks/src/main/scala/dagr/tasks/picard/FastqToUnmappedSam.scala

```diff
@@ -52,30 +53,23 @@ class FastqToUnmappedSam(fastq1: PathToFastq,
                          library: Option[String] = None,
                          readGroupName: Option[String] = None,
                          useSequentialFastqs: Boolean = false
-                          ) extends PipedTask with FixedResources {
+                          ) extends Pipeline {
```

Comment:
I think FastqToUnmappedSam should return something that is pipeable(!), but I don't think all tasks should extend `Piping`.  Metrics tasks for example could extend `PipeIn` but it seems like that would be over very limited utility since they would discard their input.